tests/keras_tests/feature_networks_tests/feature_networks/nested_networks/nested_model_multiple_inputs_test.py

```diff
@@ -35,8 +38,13 @@ class NestedModelMultipleInputsTest(BaseKerasFeatureNetworkTest):
     def __init__(self, unit_test):
         super().__init__(unit_test)
 
+    def get_fw_hw_model(self):
+        hwm = generate_test_hw_model({'weights_n_bits': 16,
+                                      'activation_n_bits': 16})
+        return generate_fhw_model_keras(name="nested_multi_inputs_test", hardware_model=hwm)
+
     def get_quantization_config(self):
```

Comment:
Since this model is heavily used in many tests, let's create a single one in tests helpers and reuse it.